docs/install/windows/install.md

```diff
@@ -0,0 +1,249 @@
+# Install ROCm (HIP SDK) on Windows
+
+To install ROCm on Windows, you can use our [quick-start guide](./install-quick.md)` or you can
```

Comment:
```suggestion
To install the HIP SDK on Windows, you can use our [quick-start guide](./install-quick.md)` or you can
```

Reply:
Suggested:

To install the HIP-SDK on Windows, use the quick-start guide (./install-quick.md) or follow the instructions below.